nacc/uds3/filters.py

```diff
@@ -1,19 +1,18 @@
-import os
 import sys
 import csv
 import re
-import fileinput
 import ConfigParser
 
 from collections import defaultdict
 
-fill_default_values = { 'nogds' : 0,
-                        'adcid' : 41,
-                        'formver' : 3 }
+fill_default_values = {'nogds': 0,
+                       'adcid': 41,
+                       'formver': 3}
+
+fill_non_blank_values = {'adcid': '41'}
+# This dictionary contains the keys used in the config
```

Comment:
Is this comment addressing line 12? I think it would be clearer if the comment was above the line it is addressing.